kingpin/actors/aws/ecs.py

```diff
@@ -753,13 +756,12 @@ def _describe_service(self, service_name):
         services = response['services']
 
         # There should never be more than one service for a given name.
-        if len(services) > 1:
+        if len(services) != 1:
```

Comment:
Wait .. this means that if there is no service, it raises an exception right? Doesn't that mean that we can't create a service if its missing?